src/index.ts

```diff
@@ -192,14 +198,55 @@ module.exports = class ServerlessPlugin {
       if (config.subscribeToExecutionLogs) {
         await addExecutionLogGroupsAndSubscriptions(this.serverless.service, aws, datadogForwarderArn);
       }
+
+      if (config.subscribeToStepFunctionLogs) {
+        const functionArn = datadogForwarderArn;
+        const resources = this.serverless.service.provider.compiledCloudFormationTemplate?.Resources;
+        const stepFunctions = Object.values((this.serverless.service as any).stepFunctions.stateMachines);
+        if (stepFunctions.length === 0) {
+          this.serverless.cli.log(
+            "Warning: subscribeToStepFunctionLogs is set to true but no step functions were found.",
```

Comment:
I think L208, L215, L231 does not need to be warning. These are expected outcome since they have enabled `subscribeToStepFunctionLogs` flag.

Reply:
Agreed for L208 and L215. I removed the `Warning:` at the beginning of the logs.

For `Warning: Setting includeExecutionData to true for step function ${stepFunction.name} so traces can be generated.`, we're modifying the customer's log settings on their step function which could affect the size of their logs and as a result increase costs. I don't think the size/cost impact is as large as it is with `level:All` but I do think it should still be called out that a modification was made to their logging settings.

Reply:
Sounds good!